apps/settings/js/panels/call/call_forwarding.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * Make sure all operations would be executed one by one.
+ *
+ * @module call/call_forwarding
+ */
+define(function(require) {
+  'use strict';
+
+  var CallSettingsTaskScheduler =
+    require('panels/call/call_settings_task_scheduler');
+  var Module = require('modules/base/module');
+  var CallUtils = require('panels/call/call_utils');
+  var CallConstant = require('panels/call/call_constant');
+  var mozSettings = window.navigator.mozSettings;
+
+  /**
+   * @class CallForwarding
+   * @returns {CallForwarding}
+   */
+  var CallForwarding = Module.create(function() {
+    // Stores current states (enabler or not) of the call forwaring reason
```

Comment:
Let's create the array based on the number of mobile connections. (just trying to make this comment visible)